cfg.py.examples

```diff
@@ -50,3 +50,19 @@ sPlayer = "/usr/bin/mpv"
 sPlayerArgs = "--title='aiostreams' --really-quiet"
 aPlayer = "/usr/bin/mpv"
 aPlayerArgs = "--title='aiostreams' --really-quiet"
+
+# Windows10 For "C:\Program Files (x86)" VLC player
+vPlayer = "start C:\PROGRA~2\VideoLAN\VLC\\vlc.exe"
```

Comment:
Please check the paths again. I wonder about `PROGRA~2` and `PROGRA~1`... Also about the double slash before `vlc.exe`

Reply:
I assume you did this because the `C:\Program Files (x86)` contains spaces. Can you try to have that in single quotes?
`"'start C:\Program Files (x86)\VideoLAN\VLC\\vlc.exe'"`
Also, have you tried if it works without the path? Like `"start vlc.exe"`

Reply:
I just tried it and I put it in single quotes also it works without the path so I have updated that as well.

Reply:
Awesome. Thank you so much...